src/oxygen/zap.py

```diff
@@ -25,8 +25,27 @@ def run_zap(self, result_file, command, check_return_code=False, **env):
         logger.info('Result file: {}'.format(result_file))
         return result_file
 
-
-    def parse_results(self, result_file):
+    def cli(self):
+        cli_interface = self.DEFAULT_CLI.copy()
+        cli_interface[('--accepted-risk-level',)] = {
+            'help': 'Set accepted risk level',
+            'default': 2,
```

Comment:
I think we leave the default value to be decided by the config file in the case of ZAP